routes/user.js

```diff
@@ -76,28 +82,260 @@ router.post('/library', function(req, res, next) {
+                            owner: playlist._id,
+                            song: s._id,
+                            translations: []
+                        })
+                        newExportSong.save(function(err) {
+                            if (err) return handleError(err)
+                        })
+                    } else {
+                      es.translations = []
+                      es.save(function(err){
+                        if (err) return handleError(err)
+                      })
+                    }
+                })
+                Song.find({
+                        $or: [{
+                            $and: [{
+                                source: s.source
+                            }, {
+                                source: {
+                                    $exists: true
+                                }
+                            }, {
+                                lang: {
+                                    $ne: s.lang
+                                }
+                            }]
+                        }, {
+                            _id: s.source
+                        }, {
+                            source: s._id
+                        }]
+                    },
+                    function(err, songs) {
+                        if (err) return handleError(err)
+                        translationss.push(songs.map((s) => s))
+                        if (i == arr.length - 1) {
+                            res.render('export1', {
+                                playlistID: playlist._id,
+                                playlistName: playlist.name,
+                                songs: playlist.songs,
+                                translationss: translationss
+                            })
+                        }
+                    })
+            })
+        })
+})
+
+router.post('/playlist/:playlist_name/export1', function(req, res, next) {
+    // console.log(req.body)
+    var playlistID = req.body.playlistID
+    var songID = req.body.songID
+    var translationID = req.body.translationID
+        // console.log(songID)
+        // res.send({msg: 'saving'})
+    ExportSong.findOne({
+        owner: playlistID,
+        song: songID
+    }, function(err, es) {
+        if (err) return handleError(err)
+        console.log(es)
+        es.translations.push(translationID)
+        es.save(function(err) {
+            if (err) {
+                res.status(400).send('error ' + err)
+            } else {
+                res.send({
+                    msg: 'saving done'
+                })
+            }
+        })
+    })
+})
+
+router.delete('/playlist/:playlist_name/export1', function(req, res) {
+    // var playlistID = ObjectId(req.body.playlistID)
+    // var songID = ObjectId(req.body.songID)
+    // var translationID = ObjectId(req.body.translationID)
+    var playlistID = req.body.playlistID
```

Comment:
If you are using es6, you can do this like:
`{playlistID, songID, translationID} = req.body;`